app/browser/reducers/pinnedSitesReducer.js

```diff
@@ -57,11 +56,8 @@ const pinnedSitesReducer = (state, action, immutableAction) => {
           action.prepend
         )
 
-        // TODO do we need this for pinned sites?
-        if (syncUtil.syncEnabled()) {
-          const newSite = state.getIn(['pinnedSites', action.siteKey])
-          state = syncUtil.updateSiteCache(state, newSite)
-        }
+        const newSite = pinnedSitesState.getSite(state, action.siteKey)
```

Comment:
`action` is immutable, so it should be `action.get('siteKey')`. I see that I already made a mistake few lines above, will that in my PR :smiley: